src/Icons/tests/Fixtures/templates/template1.html.twig

```diff
@@ -4,7 +4,7 @@
     <li id="third">{{ ux_icon('sub:check', {'data-action': 'string "with" quotes'}) }}</li>
     <li id="fourth">{{ ux_icon('sub:check', { height: 24, width: "24" }) }}</li>
     <li id="fifth"><twig:UX:Icon name="user" class="h-8 w-8" /></li>
-    <li id="sixth"><twig:ux:icon name="sub:check" aria-labelledby="foo" /></li>
+    <li id="sixth"><twig:UX:Icon name="sub:check" aria-labelledby="foo" /></li>
```

Comment:
This one is a test (the various formats) to ensure we can handle them all

Reply:
Oops, good catch